.circleci/config/commands/check-branch-name.yml

```diff
@@ -0,0 +1,14 @@
+description: >
+  Check if branch name starts with ui/ or docs/ and if so, exit.
+steps:
+  - run:
+      working_directory: ~/
```

Comment:
I don't think setting the working directory should be necessary, as this command doesn't expect any files to be present.